.github/workflows/ci.yml

```diff
@@ -48,8 +47,8 @@ jobs:
       - run: terraform validate
 
   docs:
-    # update docs after merge back to develop
-    if: github.event_name != 'pull_request' && contains('refs/heads/develop', github.ref)
+    # update docs after merge back to main
+    if: github.event_name != 'pull_request' && contains('refs/heads/main', github.ref)
```

Comment:
question: Is this really working? I think we are not allowed to push to `main` directly. And if we do, this might create an entry in the changelog which we don't want to have, right?

suggestion: Might be better to add this step to the feature branch pipeline.

Reply:
You right, but the App installed in the repo is allowed to bypass the branch protection. Is needed to update the changelog.